src/main/scala/mesosphere/marathon/state/Group.scala

```diff
@@ -143,12 +143,12 @@ object Group extends StrictLogging {
   def defaultDependencies: Set[PathId] = Set.empty
   def defaultVersion: Timestamp = Timestamp.now()
 
-  def validGroup(base: PathId, enabledFeatures: Set[String]): Validator[Group] =
+  def validGroup(base: PathId, config: MarathonConf): Validator[Group] =
```

Comment:
What is the motivation for this change?

Reply:
We now need the config.mesosRole inside the GroupValidator, not only the config.enabledFeatures